source/libraries/ContractDeployer.ts

```diff
@@ -227,7 +227,10 @@ export class ContractDeployer {
     }
 
     private async generateAddressMapping(): Promise<string> {
-        const mapping: { [name: string]: string } = {};
+        type ContractAddressMapping = { [name: string]: string };
+        type NetworkAddressMapping = { [networkId: string]: ContractAddressMapping };
```

Comment:
Any preference on potentially putting these right after the imports? Unless the type is dynamic somehow that feels more right to me than in the function

Reply:
I am keeping these local to this function only because they're only used within the function to generate some JSON.

I'm comfortable moving them if its a big concern, but it feels like this whole function could move and bring its internal types with it.